app/navigation/Navigator.scala

```diff
@@ -92,20 +92,23 @@ class Navigator @Inject()() {
       case  _  => routes.RepaymentTypeController.onPageLoad(NormalMode)
     }
 
-  private def getEntryDetails(answers: UserAnswers): Call = answers.get(CustomsRegulationTypePage) match {
-    case Some(CustomsRegulationType.UnionsCustomsCodeRegulation)  => {
-      answers.get(NumberOfEntriesTypePage).contains(NumberOfEntriesType.Single) match {
-        case true => routes.ArticleTypeController.onPageLoad (NormalMode)
-        case _=> routes.BulkFileUploadController.showFileUpload
-      }
+  private def getEntryDetails(answers: UserAnswers): Call =
+    answers.get(CustomsRegulationTypePage).contains(CustomsRegulationType.UKCustomsCodeRegulation) match {
+      case true => routes.UkRegulationTypeController.onPageLoad(NormalMode)
+      case false => routes.ArticleTypeController.onPageLoad(NormalMode)
     }
-    case Some(CustomsRegulationType.UKCustomsCodeRegulation) => {
-      answers.get(NumberOfEntriesTypePage).contains(NumberOfEntriesType.Single) match {
-        case true => routes.UkRegulationTypeController.onPageLoad (NormalMode)
-        case _=> routes.BulkFileUploadController.showFileUpload
-      }
+
+  private def getArticleType(answers: UserAnswers) : Call =
```

Comment:
Can we refactor this? Maybe just match on answers.get(NumberOfEntriesTypePage) and check the NumberOfEntriesType.